standalone/pruntime/enclave/src/lib.rs

```diff
@@ -979,6 +974,8 @@ fn init_runtime(input: InitRuntimeReq) -> Result<Value, Value> {
         return Err(json!({"message": "Already initialized"}));
     }
 
+    env_logger::init();
```

Comment:
@h4x3rotab Should I remove this init then?

Reply:
No. The enclave world and the outside world are different, and they has to be initialized twice

Reply:
I think the only shared part is the stdout/err channel